### ISSUE TITLE
[Bug]: Afval Wijzer doesn't support ICS anymore

### ISSUE THREAD

**Issue by rugebiker** [mampfes/hacs_waste_collection_schedule#3105]
### I Have A Problem With:

A specific source

### What's Your Problem

Since some time ago, Afval Wijzer doesn't support ICS anymore. It only shows the information in their own website: https://www.mijnafvalwijzer.nl/

### Source (if relevant)

https://www.mijnafvalwijzer.nl/

### Relevant Configuration

```YAML
As an example, feel free to use in the website the following address:

Postcode: 5613JZ
Huisnummer: 25
```

### Checklist Source Error

- [X] Use the example parameters for your source (often available in the documentation) (don't forget to restart Home Assistant after changing the configuration)
- [X] Checked that the website of your service provider is still working
- [X] Tested my attributes on the service provider website (if possible)
- [X] I have tested with the latest version of the integration (master) (for HACS in the 3 dot menu of the integration click on "Redownload" and choose master as version)

### Checklist Sensor Error

- [X] Checked in the Home Assistant Calendar tab if the event names match the types names (if types argument is used)

### Required

- [X] I have searched past (closed AND opened) issues to see if this bug has already been reported, and it hasn't been.
- [X] I understand that people give their precious time for free, and thus I've done my very best to make this problem as easy as possible to investigate.

**Comment by gilaberticus** [comment]
Just created a pull request to fix this: https://github.com/mampfes/hacs_waste_collection_schedule/pull/3244

**Comment by 5ila5** [comment]
fixed by #3244

This will be part of the next release or is available now as master (git) version 

install master/git: Developer tools -> actions -> select `update: Install update` -> click `Coose entity` and select WCS -> select `version` and type `master` in the input field -> run the action -> restart HA


<details><summary>Older versions of HACS:</summary>
<p>
install master with HACS version < 2.0: (3-dot menu -> redownload -> change version to master -> fully restart HA (this will increase update cycle by a lot)
</p>
</details>
